### PR TITLE
Added field information of module InventoryDistanceBasedSourceSelection

### PR DESCRIPTION
**Pull request by sandipklevu**
## Purpose of this pull request

This pull request (PR) provides information about the missing fields as per the 2.3.x both(Open Source and Commerce) editions. It is part of the standard Inventory module.

**Module Name:** `Magento_InventoryDistanceBasedSourceSelectionAdminUi`

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.3/config-guide/prod/config-reference-sens.html
- https://devdocs.magento.com/guides/v2.3/config-guide/prod/config-reference-most.html#inventory-paths

## Links to Magento source code

-  https://github.com/magento/inventory/blob/83fc7de7a53ae60ebc3115d62cd4119c725204f2/InventoryDistanceBasedSourceSelectionAdminUi/etc/adminhtml/system.xml#L8

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by rogyar** [comment]
Example from the codebase: 
https://github.com/magento/inventory/blob/83fc7de7a53ae60ebc3115d62cd4119c725204f2/InventoryDistanceBasedSourceSelectionAdminUi/etc/adminhtml/system.xml#L8

File: src/guides/v2.3/config-guide/prod/config-reference-most.md [hunk 1]
```diff
@@ -276,6 +276,9 @@ Notify for Quantity Below | `cataloginventory/item_options/notify_stock_qty` | <
 Enable Qty Increments | `cataloginventory/item_options/enable_qty_increments` | <!-- ![Not EE-only]({{ site.baseurl }}/common/images/red-x.png) --> |
 Qty Increments | `cataloginventory/item_options/qty_increments` | <!-- ![Not EE-only]({{ site.baseurl }}/common/images/red-x.png) --> |
 Automatically Return Credit Memo Item to Stock | `cataloginventory/item_options/auto_return` | <!-- ![Not EE-only]({{ site.baseurl }}/common/images/red-x.png) --> |
+Provider | `cataloginventory/source_selection_distance_based/provider` | ![EE-only]({{ site.baseurl }}/common/images/cloud_ee.png) |
```
Comment (rogyar) [inline src/guides/v2.3/config-guide/prod/config-reference-most.md:279]: Hi @sandipklevu. Thank you for your collaboration. 
I would ask you to double-check the fact that the mentioned settings are available in the Commerce edition only. They are part of the standard Inventory codebase (see the link in the comments) with no mention of being a part of `ScalableInventory` only. 

Thank you.

Comment (sandipklevu) [reply src/guides/v2.3/config-guide/prod/config-reference-most.md]: Hi @rogyar Thanks for the suggestion. 
I have made the changes by removing the EE changes.

**Comment by dobooth** [comment]
Hi @sandipklevu Looks like there is a merge-conflict to resolve. There are have been some edits to those tables lately. Thanks!

**Comment by sandipklevu** [comment]
> Hi @sandipklevu Looks like there is a merge-conflict to resolve. There are have been some edits to those tables lately. Thanks!

@dobooth I'm able to resolve the merge conflict. Thanks

**Comment by dobooth** [comment]
running tests

**Comment by bdenham** [comment]
@rogyar Please add a whatsnew entry to your description below your Links entry. Thanks.

**Pull request merged**